tutorials/admin/services/services.md

```diff
@@ -1 +1,75 @@
-# services
+# Services
+
+The Marble software can be configured to provide several services to the users of your node. Services are user-facing
+tools that largely fall under the following categories:
+
+- serving and cataloging data hosted on your server
+- interactive development environment
+- remote processing tools
+
+(enable)=
+## Enable an included service
+
+You can pick and choose which services you would like to offer your users by enabling them via the `env.local` file.
+
+The Marble software comes with several services already included. The list of services already included can be found
+in the 
+[Birdhouse documentation](https://birdhouse-deploy.readthedocs.io/en/latest/birdhouse/components/README.html).
+
+This documentation explains what each service is used for and how to enable and configure it.
```

Comment:
You mean the document linked above in the preceding paragraph? If so, then I think this remark shouldn't be on a separate paragraph, but instead should be incorporated within the other paragraph.